superx/routing.py

```diff
@@ -17,4 +17,6 @@ def register():
 def logged_in():
    return home.logged_in()
 
-   
\ No newline at end of file
+# @app.route("/logout")
+# def logout():
+#     return signup.logout()
```

Comment:
remove code you aren't using.
Last line should be blank (no spaces or tabs, either)